test/ranges.jl

```diff
@@ -14,9 +14,11 @@
         r = 4:9
         @test r === range(start=first(r), stop=last(r)                  )
         @test r === range(start=first(r),               length=length(r))
-        # the next one uses ==, because it changes the eltype
-        @test r  == range(start=first(r), stop=last(r), length=length(r))
         @test r === range(                stop=last(r), length=length(r))
+        @test r === range(first(r),       last(r))
+        # the next ones use ==, because it changes the eltype
+        @test r == range(first(r),       last(r),      length(r))
+        @test r == range(start=first(r), stop=last(r), length=length(r))
```

Comment:
```suggestion
        @test r === range(                stop=last(r), length=length(r))
        @test r === range(      first(r),      last(r)                  )
        # the next ones use ==, because it changes the eltype
        @test r  == range(      first(r),      last(r),        length(r))
        @test r  == range(start=first(r), stop=last(r), length=length(r))
```

@jw3126 did a nice job of lining similar aspects from each line. Perhaps we should maintain that style.